aws/internal/service/guardduty/waiter/waiter.go

```diff
@@ -57,3 +65,36 @@ func AdminAccountNotFound(conn *guardduty.GuardDuty, adminAccountID string) (*gu
 
 	return nil, err
 }
+
+// PublishingDestinationCreated waits for GuardDuty to return Publishing
+func PublishingDestinationCreated(conn *guardduty.GuardDuty, destinationID, detectorID string) (*guardduty.CreatePublishingDestinationOutput, error) {
+	stateConf := &resource.StateChangeConf{
+		Pending:    []string{guardduty.PublishingStatusPendingVerification},
+		Target:     []string{guardduty.PublishingStatusPublishing},
+		Refresh:    guardDutyPublishingDestinationRefreshStatusFunc(conn, destinationID, detectorID),
+		Timeout:    PublishingDestinationCreatedTimeout,
+		MinTimeout: PublishingDestinationCreatedMinTimeout,
```

Comment:
Typically we should be able to skip declaring the `MinTimeout` for the waiters. 👍